test/ids-swaplist/ids-swaplist-func-test.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * @jest-environment jsdom
+ */
+import IdsSwapList from '../../src/components/ids-swaplist/ids-swaplist';
+
+const HTMLSnippets = {
+  VANILLA_COMPONENT: (
```

Comment:
```suggestion
  SWAPLIST_COMPONENT: (
```

Would this be more clear?